app/elements/add-question/add-question.html

```diff
@@ -33,9 +33,10 @@
     <paper-card heading="Add Question" hidden$="{{computeFormHidden(user)}}">
       <form id="form" is="iron-form" method="post" action="/form/handler">
         <div class="card-content">
-          <paper-input name="question" label="Question:" required></paper-input>
-          <paper-textarea name="answer" label="Answer:" required></paper-textarea>
-          <input name="user" type="hidden" value="[[user.uid]]">
+          <paper-input id="question" name="question" label="Question:" required></paper-input>
+          <paper-textarea id="answer" name="answer" label="Answer:" required></paper-textarea>
+          <paper-questions-tags-input id="tags"></paper-questions-tags-input>
```

Comment:
It's better to call i `question-tags-input` rather than `paper-questions-tags-input`.

It's needed to import it on the top of this element